src/attack_flow_builder/src/assets/builder.config.validator.ts

```diff
@@ -11,6 +11,8 @@ import {
 
 class AttackFlowValidator extends DiagramValidator {
 
+    static STIXregex = /^(?:[a-z][a-z0-9-]+[a-z0-9]--[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[1-5][0-9a-fA-F]{3}-[89abAB][0-9a-fA-F]{3}-[0-9a-fA-F]{12}|Null)$/i
```

Comment:
This is great. However, now that the regex is case-insensitive you can remove the upper-case variants from the regex:
- `[0-9a-fA-F]` -> `[0-9a-f]`
- `[89abAB]` -> `[89ab]`
- `Null` -> `null`

Reply:
Done!